gluecodium/src/test/resources/smoke/errors/output/dart/ffi/ffi_smoke_ErrorsInterface.cpp

```diff
@@ -43,24 +50,33 @@ class smoke_ErrorsInterface_Proxy : public ::smoke::ErrorsInterface {
     }
     gluecodium::Return<void, ::smoke::Payload>
     method_with_payload_error() override {
-        int64_t _error = (*reinterpret_cast<int64_t (*)(uint64_t)>(f3))(token
-        );
+        int64_t _error;
+        dispatch([&]() { _error = (*reinterpret_cast<int64_t (*)(uint64_t)>(f3))(token
+        ); });
     }
     gluecodium::Return<std::string, ::smoke::Payload>
     method_with_payload_error_and_return_value() override {
         FfiOpaqueHandle _result_handle;
-        int64_t _error = (*reinterpret_cast<int64_t (*)(uint64_t, FfiOpaqueHandle*)>(f4))(token,
+        int64_t _error;
+        dispatch([&]() { _error = (*reinterpret_cast<int64_t (*)(uint64_t, FfiOpaqueHandle*)>(f4))(token,
             &_result_handle
-        );
+        ); });
     }
 private:
     uint64_t token;
+    int32_t isolate_id;
```

Comment:
make token and isolate_id  const?

Reply:
Not a big improvement, so I created a ticket to handle it later #233.